src/test/java/com/echoserver/test/EchoServerTest.java

```diff
@@ -1,13 +1,35 @@
-
 package com.echoserver.test;
 
 import com.echoserver.EchoServer;
 import org.junit.Test;
 
+import java.io.*;
+
 public class EchoServerTest {
+    private EchoServer server = null;
+    private StringWriter out = null;
+
     @Test
-    public void test() {
-        EchoServer server = new EchoServer("my server");
-        assert server.getName().equals("my server");
+    public void exitTheEchoServerWithoutOutput() throws IOException {
+        initialiseEcheServerWithInput("exit\n");
+        server.run();
+
+        assert out.toString().equals("");
```

Comment:
For info, there are richer asserts available with Junit. You can say Assert.assertEquals or Assert.assertTrue for example. 

http://junit.sourceforge.net/junit3.8.1/javadoc/junit/framework/Assert.html